src/CONST.ts

```diff
@@ -1412,6 +1412,11 @@ const CONST = {
             MAKE_MEMBER: 'makeMember',
             MAKE_ADMIN: 'makeAdmin',
         },
+        DISTANCE_RATES_BULK_ACTION_TYPES: {
```

Comment:
NAB I feel like we're defining this over and over, maybe we should just have a general `BULK_ACTION_TYPES` and use for all pages, since they all have the same bulk actions

Reply:
If they're really the same then I think doing what you've proposed makes sense

Reply:
I havent seen other PRs but looking at what we have in codebase there are the ones Ive added and the ones for members which we can see just above and they seem to be different. How would you want to unify them?